app/views/admin/tags/_form.html.haml

```diff
@@ -0,0 +1,56 @@
+#notice{class: flash[:type]}
+  = flash[:message]
+
+= form_for [:admin, @tag] do |f|
+  #form_elements_div
+    .form_group.element_div
+      = f.label :name
+      = f.text_field :name
+
+    .form_group.element_div
```

Comment:
Tags don't have all these fields.

Reply:
We should probably leave that in for now, because of the tasks for the next iteration is to create the tags form.